payloads/compute.go

```diff
@@ -451,3 +451,19 @@ func NewCiaoEvents() (events CiaoEvents) {
 	events.Events = []CiaoEvent{}
 	return
 }
+
+// HTTPReturnCodeData represents the data of a call to be return
+// when explicit data of the call as error messages
```

Comment:
Could we call that one HTTPErrorData ?
And the comment should be something along those lines:
```
// HTTPErrorData represents the HTTP response body for
// a compute API request error.
```